tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -0,0 +1,559 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
strange to me that this is a copy-paste of `main` and appears as new code in the new file file instead of renaming it? Can you please just rename `main.py`?

Reply:
We still have main.py, so it is not renaming.